Python/Product/PythonTools/PythonTools/Repl/BasePythonReplEvaluator.cs

```diff
@@ -444,10 +444,18 @@ public DisconnectedException(string message)
                     return "Cannot attach to debugger when already attached.";
                 }
 
+                var pyService = _eval._serviceProvider.GetPythonToolsService();
+
+                // Only pass options that make sense for the REPL.
+                var debugOptions = PythonDebugOptions.None;
+                if (pyService.DebuggerOptions.DebugStdLib) {
+                    debugOptions |= PythonDebugOptions.DebugStdLib;

```

Comment:
I would assume that anyone adding things to PythonDebugOptions would do a Find All References on it to see all uses, at the minimum. So comments are for cases that are not covered by that (e.g. for stuff in .py files).